renderdoc/driver/shaders/spirv/spirv_debug.cpp

```diff
@@ -2840,6 +2840,14 @@ void ThreadState::StepNext(ShaderDebugState *state, const rdcarray<ThreadState>
           set0001(result);
         }
       }
+      else if(texType & DebugAPIWrapper::Buffer_Texture)
```

Comment:
For ease of review and rebasing/merging can you remove this fix's code from this PR? It shouldn't conflict since it doesn't seem to be mesh-shader specific.

Reply:
Yes will do